nuxeo-admin-console-web/angular-app/src/app/features/sub-features/generic-multi-feature-layout/components/document-tab/document-tab.component.scss

```diff
@@ -15,9 +15,32 @@
     font-size: 16px;
     bottom: 3px;
   }
+  &__button.video-renditions {
+    left: 0;
+    bottom: 0;
+  }
+  mat-form-field.hy-form-field {
+    margin-top: 10px;
+    max-width: 350px;
+    width: 80%;
+  }
+  mat-form-field.hy-form-field.video-renditions {
```

Comment:
leverage features of scss. 
mat-form-field {
&.hy-form-field {...}
&.hy-form-field.video-renditions {...}